src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -57,277 +57,274 @@
+  "tile.blockhuttownhall.abandonaddofficer": "If you want to add yourself as an officer to the abandoned colony (to retain control), click:",
   "tile.blockhuttownhall.deletemessagelink": "[DELETE]",
   "tile.blockhuttownhall.abandonmessagelink": "[ABANDON]",
   "tile.blockhuttownhall.addofficermessagelink": "[ADD AS OFFICER]",
 
   "item.minecolonies.ancienttome": "Ancient Tome",
   "item.minecolonies.chiefsword": "Chief Sword",
-  "item.minecolonies.scepterGuard": "Guard-scepter",
+  "item.minecolonies.scepterGuard": "Guard Scepter",
   "item.minecolonies.iron_scimitar": "Iron Scimitar",
   "item.minecolonies.pharaoscepter": "Pharao Scepter",
   "item.minecolonies.firearrow": "Fire Arrow",
   "item.minecolonies.banner_rally_guards": "Rallying Banner",
-  "item.minecolonies.banner_rally_guards.activated": "You have rallied your army. %d guards are answering your call!",
-  "item.minecolonies.banner_rally_guards.activated.noguards": "You have rallied your army ... of 0 guards. Good luck!",
-  "item.minecolonies.banner_rally_guards.deactivated": "You have dismissed your army. Your guards will return to regular duty.",
-  "item.minecolonies.banner_rally_guards.outofrange": "Your guards will not rally so far away to your position, they are sworn to protect their families! The disappointed guards are going home now.",
-
-  "entity.worker.inventoryfullchestfull": "Please make space in my building chest. I am not able to dump my inventory.",
-
-  "entity.builder.messagebuildstart": "Started building %s",
-  "entity.builder.messagebuildcomplete": "Finished building %s",
-  "entity.builder.messagebuildcomplete_generic": "Builder %s finished building %s",
-  "entity.builder.messageremovalcomplete": "Finished deconstructing building %s",
-  "entity.builder.messagebuildtoohigh": "%s is too high, Please lower the building below Y 256.",
-  "entity.builder.messagebuildtoolow": "%s is too low, Please raise the building above Y 0.",
-  "entity.builder.messageneedmaterial": "I need %s (Amount: %d)",
-  "entity.builder.messagebuildernecessary": "Requiring a builder's hut of level %d in order to build or upgrade this building.",
+  "item.minecolonies.banner_rally_guards.activated": "You have rallied your army. %d Guards are answering your call!",
+  "item.minecolonies.banner_rally_guards.activated.noguards": "You have rallied your army...of 0 Guards. Good luck!",
+  "item.minecolonies.banner_rally_guards.deactivated": "You have dismissed your army. Your Guards will return to regular duty.",
+  "item.minecolonies.banner_rally_guards.outofrange": "Your Guards will not rally so far away, they are sworn to protect their families! The disappointed Guards are going home now.",
+
+  "entity.worker.inventoryfullchestfull": "Please make space in my chests/racks. I am not able to dump my inventory.",
+
+  "entity.builder.messagebuildstart": "Builder %s started building %s.",
```

Comment:
I think these are appended to the builder name today, so this is effectively what you get pre-change, except with weird capitalization. Let me look them up.

Reply:
Yeah, this does special formatting prepending the message. If you are in a colony other than the one sending it it also includes the colony name.